linera-base/src/prometheus_util.rs

```diff
@@ -38,6 +38,20 @@ pub fn register_histogram_vec(
     register_histogram_vec!(histogram_opts, label_names).expect("Histogram can be created")
 }
 
+/// Construct the bucket interval starting from a value and an ending value.
+pub fn bucket_interval(start_value: f64, end_value: f64) -> Option<Vec<f64>> {
+    let quot = end_value / start_value;
+    let factor = 3.0_f64;
+    let count_approx = quot.ln() / factor.ln();
+    let count = count_approx.round() as usize;
```

Comment:
Do we care if we exceed `end_value`? If we do then maybe we should use `floor()` instead 🤔
```suggestion
    let count = count_approx.floor() as usize;
```